keyboards/waldo/waldo.h

```diff
@@ -0,0 +1,51 @@
+#ifndef WALDO_H
+#define WALDO_H
+
+#include "quantum.h"
+
+#define KEYMAP_ANSI( \
```

Comment:
We're moving away from using `KEYMAP` and are moving towards `LAYOUT`. The reason is that this is LAYOUT that contains a keymap, not a keymap that contains a keymap. You can actually look at our existing default names for inspiration

https://github.com/qmk/qmk_firmware/tree/master/layouts/default